src/ProjectTemplates/test/WorkerTemplateTest.cs

```diff
@@ -22,7 +22,6 @@ public WorkerTemplateTest(ProjectFactoryFixture projectFactory, ITestOutputHelpe
         public ITestOutputHelper Output { get; }
 
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/19716")]
```

Comment:
Linked issue has been closed with a fix that we believe was affecting this test.